airflow/utils.py

```diff
@@ -775,3 +786,33 @@ def logger(self):
             self._logger = logging.root.getChild(self.__class__.__module__ + '.' +self.__class__.__name__)
             return self._logger
 
+
+def get_sqla_engine(pool_size=5, pool_recycle=3600):
+    """
+    Get a sqlalchemy engine object, pass pool_size=0 to get an engine
+    without a connection pool.
+    """
+    CONN = settings.SQL_ALCHEMY_CONN
+    engine_args = {}
+    if 'sqlite' not in CONN and pool_size:
+        # Engine args not supported by sqlite
```

Comment:
I'm curious why NullPool for sqllite instead of the default SingletonThreadPool: http://docs.sqlalchemy.org/en/latest/dialects/sqlite.html